cardano-api/src/Cardano/Api.hs

```diff
@@ -48,6 +48,11 @@ module Cardano.Api
 
   , queryFilteredUTxOFromLocalState
   , queryPParamsFromLocalState
+
+  -- Delegation Certificate Related
+  , shelleyDeregisterStakingKey
+  , shelleyDelegateStake
+  , shelleyRegisterStakingKey
```

Comment:
Technically we register a "stake address", not a "staking key". Stake addresses can use either a single key or a script. We're only covering the key case so far.